### DISCUSSION TITLE
5.0.20版本为什么padding: auto在我们本地和g2官网生效，我们部署到生产环境就不生效了？5.0.18正常

### DISCUSSION THREAD

**Discussion by ChristmasFox** [antvis/G2#5443] [Q&A]
我们已经确保本地和生产的依赖一致 都清除缓存重新install了
5.0.18及之前都正常
<img width="611" alt="image" src="https://github.com/antvis/G2/assets/68337739/e7354907-cab9-4240-b243-9842e2eaabc7">
<img width="538" alt="image" src="https://github.com/antvis/G2/assets/68337739/e300ad80-068e-4783-a2b1-2c93e9c20b30">

**Comment by hustcc** [comment]
看截图确实有问题。稍等，我看看，你这边先写死版本。

**Reply by hustcc** [reply]
回顾了一下 commit，似乎没有这相关的提交，而且确实官网测试没问题，你要不在你项目中调试看看有没有更多的信息。

**Reply by ChristmasFox** [reply]
好 我们暂时先用5.0.18版本 等后续继续调试看看
